models/dataset.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 14 20:36:19 2021
+
+@author: NLEGENDN
+"""
+import os
+import numpy as np
+import torch
+from torch.utils.data import DataLoader, random_split
+import functools
+import h5py
+from PIL import Image
+CUDA = torch.cuda.is_available()
+
+if CUDA:
+    CRYO_TRAIN_VAL_DIR = os.getcwd()+"/Cryo/VAE_Cryo_V3/Data/"
+else:
+    CRYO_TRAIN_VAL_DIR = os.getcwd()+"\\Data\\"
+
+
+KWARGS = {'num_workers': 1, 'pin_memory': True} if CUDA else {}
+
+
+def open_dataset(path, new_size, D3):
+    """
+    Parameters
+    ----------
+    path : string path.
```

Comment:
Docstring is almost perfect! Not quite though, please look at the contributing guidelines to finish to clean it

Reply:
Finish cleaning the docstring